packages/tasks/src/snippets/inputs.ts

```diff
@@ -31,6 +31,12 @@ const inputsTableQuestionAnswering = () =>
 		}
 	}`;
 
+const inputsDocumentQuestionAnswering = () =>
```

Comment:
Note that this same input and snippet will work for Visual Question Answering, so we should reuse them if possible